src/Grpc/test/InteropTests/InteropTests.csproj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <ContainsFunctionalTestAssets>true</ContainsFunctionalTestAssets>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+
+    <!-- https://github.com/dotnet/aspnetcore/issues/24182 -->
+    <BuildHelixPayload>false</BuildHelixPayload>
```

Comment:
@JamesNK and @JunTaoLuo do you remember why we needed to do this❔ I'm wondering if the real problem is the same as what I hit initially in my dougbu/update.branding.6.0 branch, namely the lack of the following for tests that **require** the just-built packages (including the newest shared Fx):
``` msbuild
<TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
```

Reply:
No there was an issue with the APIs that were called. @JamesNK fixed it in https://github.com/dotnet/aspnetcore/pull/24824. Seems to me that disabling the tests should not be merged forward.

Reply:
nit: Merging this doesn't disable the tests, it just means they run on build agents instead of Helix. But, if it doesn't need to be done in master, great 😺